tools/driver/cling.cpp

```diff
@@ -106,6 +106,20 @@ int main( int argc, char **argv ) {
     Interp.loadFile(Lib);
 
   cling::UserInterface Ui(Interp);
+
+  // Get the home directory
+  llvm::SmallString<32> HomeDir;
+  llvm::sys::path::home_directory(HomeDir);
+
+  // Look for .cling_init in the home directory and if found process it like a cling script
+  llvm::SmallString<128> ClingInit(HomeDir.c_str());
```

Comment:
Why do we need a second variable? We should just append to the variable filled by `home_directory`

Reply:
I felt it was more expressive that way but your way is more efficient